drivers/SmartThings/zwave-sensor/src/init.lua

```diff
@@ -67,14 +67,14 @@ local function do_configure(driver, device)
   end
 end
 
-local initial_events_map = {
-  [capabilities.tamperAlert.ID] = capabilities.tamperAlert.tamper.clear(),
-  [capabilities.waterSensor.ID] = capabilities.waterSensor.water.dry(),
-  [capabilities.moldHealthConcern.ID] = capabilities.moldHealthConcern.moldHealthConcern.good(),
-  [capabilities.contactSensor.ID] = capabilities.contactSensor.contact.closed(),
-  [capabilities.smokeDetector.ID] = capabilities.smokeDetector.smoke.clear(),
-  [capabilities.motionSensor.ID] = capabilities.motionSensor.motion.inactive()
-}
+-- local initial_events_map = {
+--   [capabilities.tamperAlert.ID] = capabilities.tamperAlert.tamper.clear(),
+--   [capabilities.waterSensor.ID] = capabilities.waterSensor.water.dry(),
+--   [capabilities.moldHealthConcern.ID] = capabilities.moldHealthConcern.moldHealthConcern.good(),
+--   [capabilities.contactSensor.ID] = capabilities.contactSensor.contact.closed(),
+--   [capabilities.smokeDetector.ID] = capabilities.smokeDetector.smoke.clear(),
+--   [capabilities.motionSensor.ID] = capabilities.motionSensor.motion.inactive()
+-- }
```

Comment:
Did the luacheck comment these out rather than remove them or was that you?

Reply:
I commented those out because that code is currently unused, but will be re-enabled after migration is done so that newly-joined devices once again emit synthetic events.

Reply:
ahh okay, forgot about this.